docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/unified/index.md

```diff
@@ -0,0 +1,273 @@
+| `enable_yauaa`            | Flag to include the [YAUAA enrichment](/docs/enriching-your-data/available-enrichments/yauaa-enrichment/index.md) data in the models.                                                                                                                                                                                                                                            | `false`                   |
+| `enable_browser_context`            | Flag to include browser context data in the models.                                                                                                                                                                                                                                            | `false`                   |
+| `enable_mobile_context`            | Flag to include mobile context data in the models.                                                                                                                                                                                                                                            | `false`                   |
+| `enable_geolocation_context`            | Flag to include the geolocation data in the models.                                                                                                                                                                                                                                            | `false`                   |
+| `enable_application_context`            | Flag to include the app context data in the models.                                                                                                                                                                                                                                            | `false`                   |
+| `enable_screen_context`            | Flag to include the mobile screen data in the models.                                                                                                                                                                                                                                            | `false`                   |
+| `enable_app_error_event`            | Flag to include the mobile app error data in the models.                                                                                                                                                                                                                                            | `false`                   |
+| `enable_deep_link_context`            | Flag to include the deep link context data in the models.                                                                                                                                                                                                                                            | `false`                   |
```

Comment:
Not for now, but once the web/mobile PR is done we might want to link to those pages here?